lib/pbench/server/api/resources/query_apis/datasets/datasets_contents.py

```diff
@@ -193,8 +196,12 @@ def postprocess(self, es_json: JSON, context: CONTEXT) -> JSON:
                     }
                 ]
             }
-
         """
+        count = int(es_json["hits"]["total"]["value"])
+        if count == 0:
+            self.logger.info("No data returned by Elasticsearch")
```

Comment:
This message is going to be useless because it has no context about why this happened, or in what context. I know, we have a lot of messages like this that were helpful during debugging and never got removed, but at some point we need to stop doing it. If it's not something that's going to be useful long-term for diagnosing server problems, we shouldn't be merging it. (And if it's important enough to keep, we're going to need more information, including at least the query parameters and the run_id.)

In this case, I don't think that's particularly useful ... if someone complains that their TOC query failed when it shouldn't have, this message provides no additional information. (We'll need access to the Elasticsearch instance as well as the query parameters to analyze further.)